src/main/java/robot/subsystems/elevator/Elevator.java

```diff
@@ -0,0 +1,198 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2017-2018 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package robot.subsystems.elevator;
+
+import com.ctre.phoenix.motorcontrol.*;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.ctre.phoenix.motorcontrol.can.VictorSPX;
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj.command.Subsystem;
+import robot.subsystems.elevator.commands.JoystickElevatorCommand;
+
+/**
+ * Subsystem of the continuous Elevator
+ */
+public class Elevator extends Subsystem {
+
+    private double setpoint;
+    /* pid slots for the four states: up and down on the first level and up and down on the second level of the cascade*/
+    private final int TALON_BOTTOM_UP_PID_SLOT = 0;
+    private final int TALON_BOTTOM_DOWN_PID_SLOT = 1;
+    private final int TALON_TOP_UP_PID_SLOT = 2;
+    private final int TALON_TOP_DOWN_PID_SLOT = 3;
+    
+    private final VictorSPX victorMotor = new VictorSPX(Ports.victorMotor);
```

Comment:
Change this to simply "victor". Calling it "motor" is actually wrong, because Victor SPX is a motor controller and not a motor. Sorry for being Katzuv